app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -305,16 +306,26 @@ private void takeAddToHomeScreenScreenshot() throws UiObjectNotFoundException {
         Screengrab.screenshot("AddtoHSDialog");
 
         TestHelper.AddtoHSCancelBtn.click();
+        TestHelper.pressBackKey();
```

Comment:
I still do not understand why we press back here now?

Reply:
you're right, I was thinking about keyboard dismissing, but i shouldn't do that if I was pressing cancel.